batch/test/test_batch.py

```diff
@@ -565,3 +565,61 @@ def test_verify_no_access_to_metadata_server(self):
         status = j.wait()
         assert status['state'] == 'Success', status
         assert "No credentialed accounts." in j.log()['main'], (j.log()['main'], status)
+
+    def test_verify_can_tcp_to_localhost(self):
+        builder = self.client.create_batch()
+        script = '''
+set -e
+nc -l -p 5000 &
+sleep 5
+echo "hello" | nc -q 1 localhost 5000
```

Comment:
I assume -q means quiet. I couldn't find it in the man pages.

Reply:
`-q` isn't in BSD/macOS netcat. It does [this](https://www.commandlinux.com/man-page/man1/nc.1.html):
```
-q seconds
    after EOF on stdin, wait the specified number of seconds and then quit. If seconds is negative, wait forever.
```

Without this argument, `netcat` will keep the connection open forever.